invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/users/search/SearchResultItemLayout.js

```diff
@@ -22,8 +23,18 @@ class SearchResultItemComponent extends Component {
     updateQueryState(currentQueryState);
   };
 
+  buildAdminLinks = (userId) => {
+    const baseQuery = `parent.access.owned_by.user:${userId}`;
+    return {
+      admin_records_html: `/administration/records?q=${baseQuery}&f=allversions`,
```

Comment:
I would like to suggest not to generate the links in the ui whenever not absolutely unavoidable, but to use the ones supplied by REST API, otherwise the code becomes very difficult to maintain